src/Utils.js

```diff
@@ -27,18 +34,94 @@ class Utils {
     return Promise.all(directories.map((directory) => fs.rmdir(directory, { recursive: true })));
   }
 
+
+  /**
+   *
+   *
+   * @static
+   * @param {} data
```

Comment:
This comment header could be updated to include the description and the `data` parameter type.